helm/cloud-provider-azure/values.yaml

```diff
@@ -30,6 +32,7 @@ cloudControllerManager:
   # minResyncPeriod: "100m"
   # nodeStatusUpdateFrequency: "10m"
   port: 10267
+  replicaCount: 1
```

Comment:
slight preference to just using the value `replicas` here (to match the standard k8s spec)

Reply:
sure, no problem